src/test/Multicall.t.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+
+import {Multicall} from "../Multicall.sol";
+import {DSTestPlus} from "./utils/DSTestPlus.sol";
+import {MockCallee} from "./mocks/MockCallee.sol";
+
+contract MulticallTest is DSTestPlus {
+  Multicall multicall;
+  MockCallee callee;
+
+  /// @notice Setups up the testing suite
+  function setUp() public {
+    multicall = new Multicall();
+    callee = new MockCallee();
+  }
+
+  /// >>>>>>>>>>>>>>>>>>>>  AGGREGATION TESTS  <<<<<<<<<<<<<<<<<<<< ///
+
+  function testAggregation() public {
+    // Test successful call
+    Multicall.Call[] memory calls = new Multicall.Call[](1);
+    calls[0] = Multicall.Call(
+        address(callee),
+        abi.encodeWithSignature("getBlockHash(uint256)", block.number)
+    );
+    (
+        uint256 blockNumber,
+        bytes[] memory returnData
+    ) = multicall.aggregate(calls);
```

Comment:
Any objection to moving all the statements like these on to one line? Keeps it shorter / easier to read imo, and is still under 120 chars wide

```suggestion
    calls[0] = Multicall.Call(address(callee), abi.encodeWithSignature("getBlockHash(uint256)", block.number));
    (uint256 blockNumber, bytes[] memory returnData) = multicall.aggregate(calls);
```